packages/calcite-components/src/components/stepper/step-bar.tsx

```diff
@@ -0,0 +1,46 @@
+import { FunctionalComponent, h } from "@stencil/core";
+
+interface StepBarProps {
+  isActive: boolean;
+  isComplete: boolean;
+  isError: boolean;
+  disabled: boolean;
+  key: number;
+}
+
+export const CSS = {
+  stepBar: "step-bar",
+  stepBarActive: "step-bar--active",
+  stepBarComplete: "step-bar--complete",
+  stepBarDisabled: "step-bar--disabled",
+  stepBarError: "step-bar--error",
+  stepBarInActive: "step-bar--inactive",
+};
+
+export const StepBar: FunctionalComponent<StepBarProps> = ({
```

Comment:
Should this file be in a `functional` folder or something so we know its a functional component? Our other functional components are in the functional folder.

Reply:
Agreed, moved to the functional folder.